config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -20,31 +20,33 @@
 
 base="$(dirname $0)"
 
-# All kubernetes-csi repos for which Prow gets configured.
-repos="
-cluster-driver-registrar
+# All kubernetes-csi repos which are part of the hostpath driver example.
+# For these repos we generate the full test matrix.
+hostpath_example_repos="
 csi-driver-host-path
-csi-release-tools
+external-attacher
+external-provisioner
+external-snapshotter
```

Comment:
We should leave snapshotter out of version skew testing.  It is still alpha and does not offer any compatibility guarantees.

Reply:
Okay. It's now tested only once, against the Kubernetes version specified in the external-snapshotter repo itself.